### PR TITLE
Handle questionnaire submission failure

### PR DESCRIPTION
**Pull request by MebinAbraham**
### What is the context of this PR?
Adds an error page for when questionnaire submission fails.

### How to review 
Create a submission failure by using the LogSubmitter and return a value of False in:
https://github.com/ONSdigital/eq-questionnaire-runner/blob/master/app/submitter/submitter.py#L9-L21

- Ensure the content is as expected.
- Clicking "submit your census again" should take you back to the page your submitted from, i.e hub/summary etc.

### Checklist

* [ ] New static content marked up for translation
* [ ] Newly defined schema content included in eq-translations repo

### CODE REVIEW

**Comment by JamesGardiner** [comment]
Translations need updating @MebinAbraham

File: templates/errors/submission-failed.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{% extends 'layouts/_base.html' %}
+
+{% set hide_signout_button = True %}
+
+{% set page_title = _("Sorry, there is a problem") %}
+
+{% block main %}
+<h1>{{ _("Sorry, there is a problem") }}</h1>
+<p data-qa="retry">{{ _("You can try to <a href='{url}'>submit your census again</a>").format(url = url_for('questionnaire.get_questionnaire'))  }}</p>
+<p>{{ _("If this problem keeps happening, please <a href='{url}'>contact us</a> for help.").format(url = contact_us_url)  }}</p>
```
Comment (JamesGardiner) [inline templates/errors/submission-failed.html:10]: kwargs shouldn't have spaces. This would fail linting if it was in a`.py` file

**Review by JamesGardiner — CHANGES_REQUESTED** [review]
See my comment re spaces round kwargs - LGTM apart from that

**Comment by MebinAbraham** [comment]
> See my comment re spaces round kwargs - LGTM apart from that

Done :)

**Review by JamesGardiner — APPROVED** [review]

File: tests/integration/questionnaire/test_questionnaire_schema_theme_thank_you.py [hunk 2]
```diff
@@ -51,3 +53,24 @@ def test_default(self):
...
+
+        # Then I should see an error page
+        self.assertStatusCode(500)
+        self.assertEqualPageTitle("Sorry, there is a problem - Census 2021")
+
+        retry_url = (
+            self.getHtmlSoup().find("p", {"data-qa": "retry"}).find("a").attrs["href"]
+        )
+        self.get(retry_url)
+        self.assertInUrl("questionnaire/summary/")
```
[the remark this answers is not in the document]
Comment (MebinAbraham) [reply tests/integration/questionnaire/test_questionnaire_schema_theme_thank_you.py:76]: Technically I saw this as testing submission. But I can make a new class to test submission then add these new tests for hub etc all into that 1 test if that makes more sense?

[the remark this answers is not in the document]
Comment (ajmaddaford) [reply tests/integration/questionnaire/test_questionnaire_schema_theme_thank_you.py:76]: This test is specifically testing the census thank you page (`TestSchemaThemeThankYou`). I think a new class to test submission makes sense.

[the remark this answers is not in the document]
Comment (MebinAbraham) [reply tests/integration/questionnaire/test_questionnaire_schema_theme_thank_you.py]: Have addressed in [`6dc683e` (#349)](https://github.com/ONSdigital/eq-questionnaire-runner/pull/349/commits/6dc683ee2d4629e18860bb63882404a085dbe76d)

File: tests/integration/questionnaire/__init__.py [hunk 3]
```diff
@@ -1,6 +1,13 @@
 from tests.integration.integration_test_case import IntegrationTestCase
 
 
+class URL:
+    HUB = "/questionnaire/"
+    FINAL_SUMMARY = "/questionnaire/summary/"
+    FINAL_CONFIRMATION = "/questionnaire/confirmation/"
```
Comment (ajmaddaford) [inline tests/integration/questionnaire/__init__.py:7]: These aren't fixed, the ids can vary between schemas.

**Review by ajmaddaford — APPROVED** [review]

**Pull request merged**